test/handshake_helper.c

```diff
@@ -544,13 +546,13 @@ static HANDSHAKE_RESULT *do_handshake_internal(
 
     server = SSL_new(server_ctx);
     client = SSL_new(client_ctx);
-    OPENSSL_assert(server != NULL && client != NULL);
+    TEST_check(server != NULL && client != NULL);
```

Comment:
i prefer not to have multiple conditions in a test.  fine if you want to leave it tho, just style.